Platform/TigerlakeBoardPkg/CfgData/CfgData_Memory.yaml

```diff
@@ -375,27 +375,11 @@
   - SaGv         :
       name         : SA GV
       type         : Combo
-      option       : 0:Disabled, 1:FixedLow, 2:FixedMid, 3:FixedHigh, 4:Enabled
+      option       : 0:Disabled, 1:FixedPoint0, 2:FixedPoint1, 3:FixedPoint2, 4:FixedPoint3, 5:Enabled
       help         : >
-                     System Agent dynamic frequency support and when enabled memory will be training at two different frequencies. Only effects ULX/ULT CPUs. 0=Disabled, 1=FixedLow, 2=FixedHigh, and 3=Enabled.
+                     System Agent dynamic frequency support and when enabled memory will be training at three different frequencies. 0:Disabled, 1:FixedPoint0, 2:FixedPoint1, 3:FixedPoint2, 4:FixedPoint3, 5:Enabled

```

Comment:
If it has only 3 frequencies,  why we have 4 fixed points in option ?

Reply:
You are right. It should be a wrong info . Just checked the latest FSP Integration Guide, it has been removed.

Reply:
ok